doc/cephfs/disaster-recovery-experts.rst

```diff
@@ -138,28 +137,30 @@ objects.
     cephfs-data-scan init
 
 Finally, you can regenerate metadata objects for missing files
-and directories based on the contents of a data pool.  This is
-a three-phase process.  First, scanning *all* objects to calculate
-size and mtime metadata for inodes.  Second, scanning the first
-object from every file to collect this metadata and inject it into
-the metadata pool. Third, checking inode linkages and fixing found
-errors.
+and directories based on the contents of a data pool. This is
+a three-phase process: 
+
+#. Scanning *all* objects to calculate size and mtime metadata for inodes.  
+#. Scanning the first object from every file to collect this metadata and
+   inject it into the metadata pool. 
+#. Checking inode linkages and fixing found errors.
 
 ::
 
     cephfs-data-scan scan_extents [<data pool> [<extra data pool> ...]]
     cephfs-data-scan scan_inodes [<data pool>]
     cephfs-data-scan scan_links
 
-'scan_extents' and 'scan_inodes' commands may take a *very long* time
-if there are many files or very large files in the data pool.
+``scan_extents`` and ``scan_inodes`` commands may take a *very long* time if
+the data pool contains many files or very large files.
 
-To accelerate the process, run multiple instances of the tool.
+To accelerate the process of running ``scan_extents`` or ``scan_inodes``, run
+multiple instances of the tool.
```

Comment:
I might change the period here to a colon, so that the reader doesn't stop at this point and try to run the tool concurrently without specifying a unique range for each, but rather reads below

Reply:
I'll trust your fresher eyes. I have replaced the period with a colon.